src/screens/CreateContactScreen.tsx

```diff
@@ -17,8 +17,11 @@ const i18n = require('../../loc');
 
 interface Props {
   navigation: CompositeNavigationProp<
-    StackNavigationProp<MainTabNavigatorParams, Route.ContactList>,
-    StackNavigationProp<MainCardStackNavigatorParams, Route.CreateContact>
+    StackNavigationProp<MainCardStackNavigatorParams, Route.CreateContact>,
+    CompositeNavigationProp<
+      StackNavigationProp<MainTabNavigatorParams, Route.ContactList>,
+      StackNavigationProp<MainCardStackNavigatorParams, Route.ScanQrCode>
+    >
```

Comment:
Awesome, could you share the part of the manual that has brought you here?

Reply:
not sure where exact but simple browse react-navigation API as remember

Reply:
any way need implementing to others similar case